configure

```diff
@@ -814,6 +814,23 @@ def host_arch_win():
   return matchup.get(arch, 'ia32')
 
 
+def libc():
+  try:
+    proc = subprocess.Popen(
+        ['ldd', '--version'],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT
+    )
+    output = proc.communicate()[0].strip().decode('utf-8')
+
+    for libc in ['glibc', 'musl']:
+        if re.search(libc, output, flags=re.IGNORECASE) is not None:
```

Comment:
`if libc in output.lower():`?

Reply:
Thanks! It's been a while since I did python.